### DISCUSSION TITLE
Unable to modify the custom script

### DISCUSSION THREAD

**Discussion by prasanth208** [netbox-community/netbox#13847] [Help Wanted!]
### NetBox version

v3.5.1

### Python version

3.10

### Steps to Reproduce

custom script is not working as expected after upgrading the version from 3.4.5 to 3.5.1
Rather couldn't able delete or insert any new script into the container via cli and gui

Add custom script into /opt/netbox/netbox/scripts
Access Scripts tab on Web UI, newly added script that was put into /opt/netbox/netbox/scripts does not show up

### Expected Behavior

new script need to be seen in the Scripts 

### Observed Behavior

Server Error
There was a problem with your request. Please contact an administrator.

**Comment by jeremystretch** [comment]
Converting this to a discussion as reproduction steps were not provided, and the NetBox version in question is out of date.

**Comment by candlerb** [comment]
> Access Scripts tab on Web UI, newly added script that was put into /opt/netbox/netbox/scripts does not show up

Netbox 3.5 has a different way of adding custom scripts, you have to upload them via the web interface (which also records them in the database). Dropping them into the scripts directory is no longer supported.

As for the "Server Error" you'd have to show what URL you were accessing when it occurred. Did it say any more than just Server Error?

If you have an error in a custom script it can be hard to find the error, see #12766. It shouldn't give Server Error. However, bad *reports* certainly did give Server Errors, see #12842 and #12841

If you have decided to use Netbox 3.5.x, then you should go straight for the latest in that train (which is v3.5.9) as you'll pick up a load of bug fixes.
